### ISSUE TITLE
Server-side deployment

### ISSUE THREAD

**Issue by RichardLitt** [RichardLitt/github-task-status#4]
In #1, one of the things to do is to make a server run that serves the images easier. Right now, it is possible to get a JSON object with the appropriate number of done or undone tasks, and a progressed.io badge based on that number. However, this isn't automatically updating anything, and this won't catch any changes -- and it needs to be done manually. 

So, what needs to happen is that there is some task out there which either hits GitHub at regularly intervals, looking for changes, or get's notified automatically when a GitHub issue changes, and then updates the badge which is sent out. The badge would then be something like: 

```
http://githubtaskstatus.io/bar/?repo=RichardLitt/github-task-status&issue=12
```

Which would then be a permanent URL, but the content would change based on the content of that issue in that repo, provided as arguments. 

To do:
- [x] Figure out if you can automatically get updates when a GitHub issue is edited
- [x] Figure out how to get a cronjob running on a GitHub server, otherwise
- [ ] Decide if we'll serve our own images, or serve a progressed.io image that is linked to that issue, and how to do this
- [ ] Buy the domain and host this process on Heroku or something. 

I would love any advice on this, as this is a lot to ask. @jbenet @diasdavid @dignifiedquire

**Comment by ghost** [comment]
You can use Github's web hooks! That's the same mechanism which triggers e.g. Travis CI builds.

**Comment by RichardLitt** [comment]
Good call. Looks like I can get pinged when an issue is [edited](https://developer.github.com/v3/activity/events/types/#issuesevent). So, that makes that easier. Now I just need to figure out how to get this on a server?
